ExeterTeam25/group-software-project
# Image analysis

To Test:
- login 
- complete a challenge 
- upload an NSFW image (offensive images (swastika for example), porn, etc.)
- the image should not be uploaded and display a message saying the image was against Terms of service
- upload a SFW image
- it should upload without problems
- do the same for a profile picture
- (Please don't spam this function as we have a limited number of api calls)

Details:
- added image analysis for uploading photos for compleating challenges and profile pics
